tests/unit/testResources/kafkaCluster.ts

```diff
@@ -20,7 +24,7 @@ export const TEST_CCLOUD_KAFKA_CLUSTER: CCloudKafkaCluster = CCloudKafkaCluster.
   region: TEST_CCLOUD_REGION,
   bootstrapServers: `SASL_SSL://pkc-abc123.${TEST_CCLOUD_REGION}.${TEST_CCLOUD_PROVIDER}.confluent.cloud:443`,
   uri: `https://pkc-abc123.${TEST_CCLOUD_REGION}.${TEST_CCLOUD_PROVIDER}.confluent.cloud:443`,
-  environmentId: TEST_CCLOUD_ENVIRONMENT.id,
+  environmentId: TEST_CCLOUD_ENVIRONMENT_ID,
```

Comment:
Unrelated to this PR; TS hollering that the `TEST_CCLOUD_ENVIRONMENT.id` wasn't an `EnvironmentID` branded string type

Reply:
Yeah, am unsure why we're notified about these in a trickle.